lib/util/filterCollection.js

```diff
@@ -1,23 +1,23 @@
-const { v4: uuidv4 } = require('uuid');
+const { _ } = require('lodash');
```

Comment:
By convention, '\_' could indicate an unused positional first parameter, _lodash(js)_, _underscode(js)_ among other things.
A suggestion would be to call your util lib *lodash*, literally, and not '_'

Reply:
solved